### DISCUSSION TITLE
Import Database Error

### DISCUSSION THREAD

**Discussion by paul-hph** [planetscale/discussion#138] [FAQs]
I would like to import my existing database via the webinterface.  I am getting the error: 

**Could not connect to database**
Error checking server configuration: expected gtid mode to be ON, is instead [OFF]

Is there any other way to import a dB? I was able to import an SQL file to branch of `production`. But i dont see any options to bring the content to the main branch.

**Comment by Phanatic** [comment]
Hi @paul-siteway,

Which database server are you using to host your existing database?
PlanetScale needs GTID mode to be turned on because we replicate any changes to the source database after an initial data copy so we don't have any data loss. We don't support other modes of data import as of right now and would love to work with you to make you successful. 

Is there any way you can enable GTID mode on the database server? We can work with you to find the right set of instructions to enable GTID mode.

Thanks,
Phani

[more about GTIDs](https://dev.mysql.com/doc/refman/5.7/en/replication-options-gtids.html)

**Reply by techandmedia** [reply]
Hi,
I already enable it, but still return with the same error
```bash
MariaDB [(none)]> SET GLOBAL gtid_strict_mode = ON;
Query OK, 0 rows affected (0.000 sec)

MariaDB [(none)]> show global variables like '%gtid_strict_mode%';
+------------------+-------+
| Variable_name    | Value |
+------------------+-------+
| gtid_strict_mode | ON    |
+------------------+-------+
1 row in set (0.002 sec)
```

Also trying to connect planetscale DB using dbForge Studio, return with response "SSL Connection Error (2026)": The server hello message uses a protocol that was not recognized"

Why is it hard to connect to planetscale DB? I can easily connect to a AWS RDS with no problem

**Reply by flowoow** [reply]
Hi, I have the same issue. Would it be because it's MariaDB and not MySQL?

**Comment by paul-hph** [comment]
The Database is currently hosted on a MySQL 5.7. 
The DB Server is **managed** so i do not have full control and might not be able to activate GTID. I will ask the hosting company if it is possible to activate it.
